docs/UserGuide.md

```diff
@@ -110,9 +110,12 @@ Parameters:
 </box>
 
 Restrictions:
-* The maximum number of characters of a volunteer name is 30.
 * The email must be in a valid format.
-* The phone number must be a valid 8-digit Singapore phone number.
+* The phone number must be a 8-digit number.
+* The skills are case-sensitive (`chef` and `Chef` are considered different skills).
+* Duplicate volunteers cannot be added to the volunteer list (case-sensitive).
```

Comment:
why cannot j follow the skills - `The name is case-sensitive` but ok this way works too

Reply:
changed